binpac/binpac.spec

```diff
@@ -58,8 +56,11 @@ rm -rf $RPM_BUILD_ROOT
 %files devel
 %{_includedir}/binpac*
 %{_libdir}/libbinpac.a
+%{_libdir}/libbinpac.so*
 
 %changelog
+* Thu Aug 22 2019 Bradford Dabbs <brad@dabbs.io> 1.1-2
```

Comment:
This version should be `0.54-1` following the `version`-`release` pattern. Yes...mine below is also wrong.

Reply:
Fixed